src/protocols/http/hpr_roaming_utils.erl

```diff
@@ -0,0 +1,128 @@
+%%%-------------------------------------------------------------------
+%%% @author jonathanruttenberg
+%%% @copyright (C) 2022, <COMPANY>
+%%% @doc
+%%%
+%%% @end
+%%% Created : 17. Sep 2022 10:56 AM
+%%%-------------------------------------------------------------------
+-module(hpr_roaming_utils).
+-author("jonathanruttenberg").
+
+-define(RESPONSE_STREAM_ETS, hpr_http_response_stream_ets).
+-include_lib("helium_proto/include/packet_pb.hrl").
```

Comment:
This should not be used anywhere now, it's old code

Reply:
OK

Reply:
It has been removed.